src/install/dependency.zig

```diff
@@ -419,7 +419,9 @@ pub const Version = struct {
                 // newspeak/repo
                 // npm:package@1.2.3
                 'n' => {
-                    if (strings.hasPrefixComptime(dependency, "npm:")) return .npm;
+                    if (strings.hasPrefixComptime(dependency, "npm:")) {
+                        return infer(dependency["npm:".len..]);
```

Comment:
Would this end up misinterpreting inputs like `npm:@scope/pkg`? :thinking:

Reply:
oh, yes.